### PR TITLE
feat(module:upload): if UploadFile manual set isImageUrl true, skip checkfile

### PR DESCRIPTION
**Pull request by jerrytang67**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by cipchk** [comment]
LGTM

File: components/upload/upload-list.component.ts [hunk 1]
```diff
@@ -205,7 +205,7 @@ export class NzUploadListComponent implements OnChanges {
       file.isUploading = file.status === 'uploading';
       file.message = this.genErr(file);
       file.linkProps = typeof file.linkProps === 'string' ? JSON.parse(file.linkProps) : file.linkProps;
-      file.isImageUrl = this.isImageUrl(file);
+      file.isImageUrl = file.isImageUrl ? true : this.isImageUrl(file);
```
Comment (cipchk) [inline components/upload/upload-list.component.ts:208]:
```suggestion
      file.isImageUrl = typeof file.isImageUrl === 'boolean' ? file.isImageUrl : this.isImageUrl(file);
```

建议使用 `typeof` 校验是否布尔类型。

Comment (jerrytang67) [reply components/upload/upload-list.component.ts]: 已修改

**Comment by cipchk** [comment]
LGTM

**Comment by jerrytang67** [comment]
why removed ? I need this feature

**Pull request closed without merging**